spec/javascripts/components/Step.spec.js

```diff
@@ -13,67 +13,75 @@ describe("Step", function() {
 		step = new Step();
 	});
 
-	describe("title", function() {
-		var defaultStepConfig = {
-			title: "Default title"
-		};
+	const fields = [
+		{name: "title", config_name: "title"},
+		{name: "summary", config_name: "summary"},
+		{name: "description", config_name: "description"},
+		{name: "assetUrls", config_name: "asset_urls"}
+	];
+
+	_.each(fields, (field) => {
+		describe(`Getter/Setters: ${field.name}`, () => {
+			var defaultStepConfig = {
```

Comment:
```suggestion
			const defaultStepConfig = {
```

Reply:
should we use const everywhere? plain js files are not transpiled so browser support might be a problem?